apps/nfw-react/src/app/project/cmp/Cart/Cart.cmp.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+import { Link } from 'react-router-dom';
+import { Nav } from '../Nav';
+
+export const Cart = ({ props }) => {
+  return (
+    <>
+      <Nav />
+
+      <div className="cart">
+        <h1 className="cart__heading">Your order:</h1>
+        <div
+          className={
+            props.order.total
+              ? 'cart__container'
+              : 'cart__container cart__container--empty'
+          }
+        >
+          {props.order.total ? (
+            <ul className="cart__list">
+              {props.order.pizza.map((p, i) => (
+                <AnimatePresence key={i}>
+                  <motion.li
+                    className="cart__item"
+                    whileTap={{ opacity: 0 }}
+                    transition={{
+                      duration: 0.8,
+                      ease: [0.04, 0.62, 0.23, 0.98],
+                    }}
+                    onClick={() =>
+                      props.deletePizzaFromCart({
+                        id: p.id,
+                        ingredients: p.ingredients,
+                      })
+                    }
+                  >
+                    <p>{props.pizzasById[p.id]?.name}</p>
```

Comment:
if that part of the code was a separate component then:
```
  <Pizza key={p.id} /> 
```
and Pizza cmp could be wired up to the store to extract pizza details + bind necessary actions
```

connect(
  (state, ownProps) => ({
    pizza: selectPizzaById(ownProps.key) // it's ID under key property
  })
  (dispatch, ownProps) => ({
    hiddenDelete: pizza => dispatch(deleteAction(pizza))
  }),
  (stateProps, dispatchProps, ownProps) => ({
    ...ownProps,
    ...stateProps,
    deleteFromCart: () => dispatchProps.hiddenDelete(stateProps.pizza)) // U're able to bind onClick={props.deleteFromChart}
 }}, 
)(Pizza)
```